components/automate-gateway/gateway/services.go

```diff
@@ -321,6 +321,7 @@ func versionedRESTMux(grpcURI string, dopts []grpc.DialOption, toggles gwRouteFe
 func muxFromRegisterMap(grpcURI string, dopts []grpc.DialOption, localEndpoints map[string]registerFunc) (*runtime.ServeMux, error) {
 	opts := []runtime.ServeMuxOption{
 		runtime.WithIncomingHeaderMatcher(headerMatcher),
+		runtime.WithMarshalerOption("application/json+pretty", &runtime.JSONPb{OrigName: true, EmitDefaults: true, Indent: "  "}),
```

Comment:
Just wondering what the Indent is for... does that add that much at each nested element?

Reply:
Yup! https://godoc.org/github.com/golang/protobuf/jsonpb#Marshaler
```
    // A string to indent each level by. The presence of this field will
    // also cause a space to appear between the field separator and
    // value, and for newlines to be appear between fields and array
    // elements.
    Indent string
```